modules/default/alert/ns-default.css

```diff
@@ -74,6 +74,7 @@
   0% {
     -webkit-transform: perspective(1000px) rotate3d(1, 0, 0, -90deg);
   }
+
```

Comment:
This is not part of the PR.

Reply:
As above; only lint fixes (nothing substantive) given change to `Gruntfile.js`. Will revert.